rosidl_pycommon/rosidl_pycommon/__init__.py

```diff
@@ -218,7 +239,10 @@ def _expand_template(template_name, **kwargs):
             'beforeInclude', name=str(template_path), file=h, locals=kwargs)
         content = h.read()
     try:
-        interpreter.string(content, str(template_path), kwargs)
+        if em_has_configuration:
+            interpreter.string(content, locals=kwargs)
+        else:
+            interpreter.string(content, str(template_path), kwargs)
```

Comment:
Sorry, I just noticed one more thing.  This looks different from both line 243, and line 193 above.  I'm wondering if we should just make this consistent with everything else:

```suggestion
            interpreter.string(content, template_path, locals=kwargs)
```

What do you think?

Reply:
https://github.com/ros2/rosidl/pull/821/commits/761d831d430a32f26548f02876e9675414f7a444